From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Sun, 16 Mar 2025 03:47:43 +0100
Subject: [PATCH] docs: Use furo theme

We don't package pydata_sphinx_theme. Return to the one pygobject used
before.
---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index d76b7714baf3..62ab1b0fb0f1 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -19,7 +19,7 @@ master_doc = 'index'
 exclude_patterns = ['_build', 'README.rst']
 
 pygments_style = 'tango'
-html_theme = "pydata_sphinx_theme"
+html_theme = "furo"
 html_show_copyright = False
 html_favicon = "images/favicon.ico"
 project = "PyGObject"
